modules/microshift-kubeconfig-hosts-ips-config.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * microshift/microshift_networking/microshift-kubeconfig.adoc
+
+:_content-type: PROCEDURE
+[id="Configuring-hosts-ips_{context}"]
+= Configuring host names and IP addresses
+
+You can configure additional host names and IP addresses on top of the defaults, which are `<node.hostnameOverride>`, `<node.nodeIP>` and `api.<dns.baseDomain>` using the remote access `kubeconfig` file.
+
+[NOTE]
+====
+You must restart {product-title} for configuration changes to be implemented.
+====
+
+.Prerequisites
+
+* You have installed the `oc` CLI tool.
+//Q: Do we actually need this or kubectl or no?
```

Comment:
If configuring additional hostnames and IP addresses then it implies using external access, so there is a need to install `oc`.